putio.py

```
@@ -177,10 +177,28 @@ def upload(cls, path, name=None, parent_id=0):
                 files = {'file': f}
             d = cls.client.request('/files/upload', method='POST',
                                    data={'parent_id': parent_id}, files=files)
-
+        logger.info(d)
         f = d['file']
         return cls(f)
 
+    @classmethod
+    def createZip(cls,file_ids):
+        d = cls.client.request('/files/zip',
+                                method='GET',params={'file_ids': file_ids})
+        if d['status'] == "OK":
+            zip_id = d['zip_id']
+            notReady = True
+	    while notReady:
+	        time.sleep(5)
+                d = cls.client.request('/zips/%i' % zip_id, method='GET')
+                #logger.info(d)
+                if d['url'] != False:
+                    notReady= False
+            return d['url']
+        else:
+            return None
```

raise exception instead of returning none